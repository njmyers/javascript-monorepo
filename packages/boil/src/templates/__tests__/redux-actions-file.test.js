import reduxActionsFile from "../redux-actions-file";

describe("it works", () => {
  test("it is a function ", () => {
    expect(typeof reduxActionsFile).toBe("function");
  });

  test("it runs without crashing", () => {
    expect(reduxActionsFile("test-component")).toMatchSnapshot();
  });
});
