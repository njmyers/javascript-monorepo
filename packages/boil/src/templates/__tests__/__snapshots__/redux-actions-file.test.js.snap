// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `
"export const createAction = (value: string) => ({
  type: \\"@TEST_COMPONENT/CREATE_ACTION\\"
});

export const deleteAction = (value: string) => ({
  type: \\"@TEST_COMPONENT/DELETE_ACTION\\"
});
"
`;
